core-web/libs/edit-content/src/lib/services/dot-edit-content.service.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { SpectatorService, SpyObject, createServiceFactory } from '@ngneat/spectator/jest';
+import { of } from 'rxjs';
+
+import { HttpClient } from '@angular/common/http';
+
+import { DotContentTypeService, DotWorkflowActionsFireService } from '@dotcms/data-access';
+
+import { DotEditContentService } from './dot-edit-content.service';
+
+import { CONTENT_TYPE_MOCK } from '../feature/edit-content/edit-content.layout.component.spec';
+
+describe('DotEditContentService', () => {
+    let spectator: SpectatorService<DotEditContentService>;
+    let dotEditContentService: DotEditContentService;
+    let contentTypeService: SpyObject<DotContentTypeService>;
+    const createService = createServiceFactory({
+        service: DotEditContentService,
+        mocks: [DotContentTypeService, DotWorkflowActionsFireService, HttpClient]
```

Comment:
Should we mock Angular's `HttpClient` or should we use the Angular `HttpTestingThing...` cc: @oidacra

Reply:
@rjvelazco @hmoreras @manuelrojas @zJaaal